tests/app.spec.js

```diff
@@ -45,17 +55,27 @@ describe('Tests for app', () => {
       expect(secondArg).toEqual(container);
     });
   });
-  describe('registerListeners', () => {
```

Comment:
Revert back to this and just add something like
```js
const spacebarEvent = new Event('keyup');
spacebarEvent.key = ' ';
document.dispatchEvent(spacebarEvent);
```
And then assert that
```
expect(actionsMock.next).toHaveBeenCalledTimes(2)
```

Reply:
You're right, it would probably be wise to split `it('register a listener for keyup events', ()...` in to three different test cases and to reset the actionsMock after running each of them